tests/test_vega.py

```diff
@@ -263,7 +256,776 @@ def test_fill_anchor_in_string(tmp_dir):
+                },
+            ],
+            "dvc_inferred_y_value",
+            [
+                {"filename": "test", "field": "acc"},
+                {"filename": "test", "field": "acc_norm"},
+            ],
+            ["rev", "dvc_inferred_y_value", "step", "field"],
+            {
+                "field": "field",
+                "scale": {"domain": ["acc", "acc_norm"], "range": [[1, 0], [8, 8]]},
+                "legend": {
+                    "symbolFillColor": "transparent",
+                    "symbolStrokeColor": "grey",
+                },
+            },
+            "datum.rev + '::' + datum.field",
+            ["rev", "field"],
+        ),
+        (
+            [
+                {
+                    "rev": "B",
+                    "dvc_inferred_y_value": "0.05",
+                    "filename": "test",
+                    "field": "acc",
+                    "step": 1,
+                },
+                {
+                    "rev": "B",
+                    "dvc_inferred_y_value": "0.1",
+                    "filename": "test",
+                    "field": "acc",
+                    "step": 2,
+                },
+                {
+                    "rev": "B",
+                    "dvc_inferred_y_value": "0.04",
+                    "filename": "train",
+                    "field": "acc",
+                    "step": 1,
+                },
+                {
+                    "rev": "B",
+                    "dvc_inferred_y_value": "0.09",
+                    "filename": "train",
+                    "field": "acc",
+                    "step": 2,
+                },
+                {
+                    "rev": "B",
+                    "dvc_inferred_y_value": "0.02",
+                    "filename": "test",
+                    "field": "acc_norm",
+                    "step": 1,
+                },
+                {
+                    "rev": "B",
+                    "dvc_inferred_y_value": "0.07",
+                    "filename": "test",
+                    "field": "acc_norm",
+                    "step": 2,
+                },
+            ],
+            "dvc_inferred_y_value",
+            [
+                {"filename": "test", "field": "acc_norm"},
+                {"filename": "test", "field": "acc"},
+                {"filename": "train", "field": "acc"},
+            ],
+            ["rev", "dvc_inferred_y_value", "step", "filename::field"],
+            {
+                "field": "filename::field",
+                "scale": {
+                    "domain": ["test::acc", "test::acc_norm", "train::acc"],
+                    "range": [[1, 0], [8, 8], [8, 4]],
+                },
+                "legend": {
+                    "symbolFillColor": "transparent",
+                    "symbolStrokeColor": "grey",
+                },
+            },
+            "datum.rev + '::' + datum.filename + '::' + datum.field",
+            ["rev", "filename::field"],
+        ),
+    ),
+)
```

Comment:
Why no tests of multiple revisions? Does it not impact the optional anchors? Should we have a test to make sure that's the case?

Reply:
I've removed the extra rev from the scatter plot test and created a separate color test